examples/fabric/v4/cloudRouterConnectivity/cloudRouter2azure/single-connection/README.md

```diff
@@ -0,0 +1,47 @@
+# ECX Fabric Layer2 Single Connection from fabric cloud router to Azure
```

Comment:
In the future, I'd like us to consider an alternative approach to providing examples: https://github.com/equinix/terraform-provider-equinix/issues/384

Reply:
I like the sound of this 👍